go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/evanphx/json-patch/v5 v5.6.0 // indirect
```

Comment:
As in KIC, I think we should bump the go version here as well.

Reply:
👍 Done